src/inc/clrconfigvalues.h

```diff
@@ -1010,6 +1010,10 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_ReadyToRun, W("ReadyToRun"), 1, "Enable/disabl
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_ReadyToRun, W("ReadyToRun"), 0, "Enable/disable use of ReadyToRun native code") // Off by default for desktop
 #endif
 
+#if defined(FEATURE_PAL) && defined(FEATURE_EVENT_TRACE)
```

Comment:
It would  be better to provide the ability to turn tracing on/off  to the tooling around collecting the events rather than provider of host, what do you think ?.

Plus what would it mean for the host to allow or disallow tracing ?